static/js/message_edit.js

```diff
@@ -305,6 +318,22 @@ function start_edit_maintaining_scroll(row, content) {
 }
 
 exports.start = function (row, edit_box_open_callback) {
+    if (Object.keys(currently_editing_messages).length === 0) {
+         $('a').on('click', function () {
```

Comment:
It seems pretty expensive to do this on any click; I think we might want to move it to the `narrow.activate / narrow.deactivate` code path...